app/diagnostics.go

```diff
@@ -731,6 +734,16 @@ func (a *App) trackPlugins() {
 		"disabled_backend_plugins":      backendDisabledCount,
 		"plugins_with_settings":         settingsCount,
 		"plugins_with_broken_manifests": brokenManifestCount,
+		"enable_antivirus":              pluginData(pluginTrackData, "antivirus"),
```

Comment:
Should we consider emitting brand new keys here instead of re-purposing the existing one? Worried it might cause unnecessary pain upstream when processing the telemetry. Maybe even introduce a `plugin_details` key, itself an object with the full plugin id as a key?

Reply:
Yes, I had proposed a name change but waiting on @jasonblais confirmation on the effects on the reports on those changes

Reply:
Discussed offline with @aaronrothschild and Chen, it was decided that option 1 makes more sense at this point given that it will not bring any breaking changes to the current reports and no other data points, besides version number, is required before the upcoming rework of telemetry. At that point, plugin config data will probably be restructured to it's own table and add any additional information as needed. 

Will update the PR to only add new entry for `version_[plugin]`.